src/store/actions.ts

```diff
@@ -0,0 +1,370 @@
+import { ActionTree } from 'vuex';
+import Irc, {
+  IrcKickEventArgs,
+  IrcMessageEventArgs,
+  Client,
+  IrcModeEventArgs,
+  IrcNickInvalidEventArgs,
+} from 'irc-framework';
+import websocket from 'irc-framework/src/transports/websocket';
+import { State } from './state';
+import parseCommands from '@/tools/parseCommands';
+import Message from '../types/message';
+import Connection from './websocket';
+import { consts } from '@/types/consts';
+import User from '@/types/user';
+import parseUsername from '@/tools/parseUsername';
+import generateNick from '../tools/generateNick';
+
+const actions: ActionTree<State, any> = {
```

Comment:
Have a line break between functions, eg

```ts
{
  first() {

  },

  second() {

  }
}
```

Reply:
Do you know how to add that as a rule?
I don't think it is possible to have actions be a class (at least not right now), so class rules wouldn't apply here, and I doubt we want all objects to have a line break.

Reply:
Yeah I think you're right. Disregard.